clients/browser/index.html

```diff
@@ -27,44 +31,73 @@
     <div class="left">
         <div class="info">
             <div class="info-title">Wallet</div>
+            <hr/>
 
-            <div class="info-block">
+            <div class="info-block init-wallet">
+
+                <div id="newWallet" >
+                    Nimiq Testnet stores your wallet in your browser<br/>
+                    Load an existing browser wallet or create a new wallet:
+                    <button class="greenbutton" onclick="startNimiq()"><strong>Create/Load Wallet</strong></button>
+                </div>
+                <div id='loadWalletFromSeed'>
```

Comment:
Use `"` for attributes instead of `'`. Also at some other locations in the code.